accessibility-checker-engine/src/v2/aria/ARIAMapper.ts

```diff
@@ -564,6 +564,11 @@ export class ARIAMapper extends CommonMapper {
             //     to that text alternative.
             //   Append the result to the accumulated text.
             if (elem.nodeName.toUpperCase() === "SLOT") {
+                //first calculate the its own text if any
```

Comment:
Can you explain this? slot elements shouldn't have children. They're filled via the assignedNodes.

Reply:
Per[ https://html.spec.whatwg.org/multipage/scripting.html#the-slot-element]), a slot element can have assigned nodes, if any, and its contents, otherwise.

Reply:
@shunguoy So shouldn't it only check inner content if the list of assigned nodes is empty?

Reply:
updated